### PR TITLE
Cluster create after api 2022-04-01

### PR DESCRIPTION
**Pull request by 25region**
### Which issue this PR addresses:

Cluster creation process after the API release

### CODE REVIEW

**Review by bennerv — DISMISSED** [review]
This will pass in PR e2e because it runs a local RP and doesn't talk to upstream.  It won't pass in an INT deploy on master because the ARM manifest has yet to be updated and deployed hence the hold label.

**Review by s-amann — APPROVED** [review]
lgtm

**Comment by bennerv** [comment]
@25region can you rebase and we can merge this?

File: pkg/util/cluster/cluster.go [hunk 1]
```diff
@@ -448,21 +448,22 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 		}
 
 		oc.Properties.WorkerProfiles[0].VMSize = api.VMSizeStandardD2sV3
+		oc.Properties.ClusterProfile.FipsValidatedModules = api.FipsValidatedModulesDisabled
```
Comment (bennerv) [inline pkg/util/cluster/cluster.go:451]: This should be removed.  We can create FIPs clusters anywhere.  

Also, can you set FIPs to enabled by default?  It should be in the api doc above on line 403

**Review by bennerv — CHANGES_REQUESTED** [review]
One change we didn't catch before.

File: pkg/util/cluster/cluster.go [hunk 2]
```diff
@@ -399,8 +399,9 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 	oc := api.OpenShiftCluster{
 		Properties: api.OpenShiftClusterProperties{
 			ClusterProfile: api.ClusterProfile{
-				Domain:          strings.ToLower(clusterName),
-				ResourceGroupID: fmt.Sprintf("/subscriptions/%s/resourceGroups/%s", c.env.SubscriptionID(), "aro-"+clusterName),
+				Domain:               strings.ToLower(clusterName),
+				ResourceGroupID:      fmt.Sprintf("/subscriptions/%s/resourceGroups/%s", c.env.SubscriptionID(), "aro-"+clusterName),
+				FipsValidatedModules: "Enabled",
```
Comment (bennerv) [inline pkg/util/cluster/cluster.go:404]: nit  for consistency.  Should be the last one.  
```suggestion
				FipsValidatedModules: api.FipsValidatedModulesEnabled,
```

**Review by bennerv — CHANGES_REQUESTED** [review]
just one final change and LGTM

**Review by bennerv — APPROVED** [review]
lgtm

**Pull request merged**